.github/workflows/weekly-release.yml

```diff
@@ -0,0 +1,36 @@
+name: Weekly check
+
+on:
+  schedule:
+    # Every Monday at 9AM
+    - cron: '0 9 * * 1'
+jobs:
+  check-kubectl-new-version:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+          fetch-depth: 0
+
+      - name: Get latest tag from the repository
+        id: repository
+        run: echo "version=$(git describe --tags --abbrev=0)" >> $GITHUB_OUTPUT
```

Comment:
I'm not understanding this combined with line 25: ain't you overwriting `version` by appending go the `$GITHUB_OUTPUT` variable?

Reply:
This is how the CI allows to store arbitrary values as the output of the step. 
In fact, this values is referenced by `steps.repository.outputs.version` (`steps.STEP_ID.outputs.OUTPUT_NAME`).

An alternative solution can be achieved by using environment variables but wouldn't bring any benefit or downside to the current solution.